src/Workspaces/Remote/ServiceHub/Services/DesignerAttribute/RemoteDesignerAttributeIncrementalAnalyzer.cs

```diff
@@ -119,7 +119,7 @@ private async Task AnalyzeProjectAsync(Project project, Document? specificDocume
                     continue;
 
                 using var memoryStream = new MemoryStream();
-                using var writer = new ObjectWriter(memoryStream);
+                using var writer = new ObjectWriter(memoryStream, canKeepObjectsAlive: false);
```

Comment:
doc.

Reply:
Will doc on the ObjectWriter constructor

Reply:
➡️ Now documented